MnemonicKit.podspec

```diff
@@ -8,10 +8,10 @@ Pod::Spec.new do |s|
   This library is originally forked from CKMnemonic: https://github.com/CikeQiu/CKMnemonic. Modifications are made for non-throwing APIs and support on OSX as well as iOS. Credit for most of this work is given to work_cocody@hotmail.com, qiuhongyang@askcoin.org.
                    DESC
 
-  s.homepage     = "https://github.com/fortmarek/MnemonicKit"
+  s.homepage     = "https://github.com/keefertaylor/MnemonicKit"
   s.license      = { :type => "MIT", :file => "LICENSE" }
   s.author             = { "Keefer Taylor" => "keefer@keefertaylor.com" }
-  s.source       = { :git => "https://github.com/fortmarek/MnemonicKit.git", :tag => "1.2.0" }
+  s.source       = { :git => "https://github.com/keefertaylor/MnemonicKit.git", :tag => "1.2.0" }
   s.source_files  = "Sources/**/*.swift",
```

Comment:
Below you've moved the source files from `Sources/MnemonicKit` to `MnemonicKit` so you'll need to update this here as well. 

OOC why move those to be in a different folder? Swift libraries seem to use either a `Source` or `Sources` folder, with either a Playground or an `Example` folder that has an example app in it. Of the top 15 repositories, only 5 seem to use this style [1]. To be clear, I don't really care about what the folder is named so don't feel the need to revert, I am just genuinely curious if you had a strong feeling about this. 

What do you think is the most idiomatic thing to do with the example app bit? Should I move it to an examples folder? Replace with a Playground? I can follow up on this in a separate PR so as not to create more work for yourself. 

[1] https://github.com/search?l=Swift&o=desc&q=swift&s=stars&type=Repositories

Reply:
It's something that I find more readable to the user who has never seen the project before. I think the `Source` folder should be now deleted and the `App` folder contents should be moved to `Example` folder. Anyway, I'd leave it to different PR, I do not mind doing it myself, but I think we should focus now to finish this one and not make it too huge 🙂